src/pint/models/stand_alone_psr_binaries/ELL1H_model.py

```diff
@@ -54,7 +54,13 @@ def __init__(self):
 
     def delayS(self):
         if set(self.fit_params) == set(["H3", "H4"]):
-            stigma = self.H4 / self.H3
+            if self.H3 == 0.0:
+                if self.H4 == 0.0:
+                    stigma = 0.0
+                else:
+                    raise ValueError("To use H4, H3 needs to be significant(H3 >= H4).")
```

Comment:
Wait, is the condition really H3 >= H4, as this claims, or is it just H3 != 0? The condition described in the exception message does not seem to be the condition the code tests for.